assignments/this.js

```diff
@@ -1,26 +1,286 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
-*
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+
+* 1. Window/Global Object Binding -
+      
+        Global Object Binding is the "default" binding of 'this' when 'this' is bound to no other object.
+
+        When 'this' is globally bound, depending on the JS Runtime Environment the value of 'this' can be
+        different things. 
+        
+        For example: console.log(this) executed purely in Node's runtime environment, will return the 'global' object..
+        
+        console.log(this) executed in Node within a node.js file however, will return the value of the 'modules.export' object
```

Comment:
Ah that's why I got an empty object when I tried to show the global object in Quokka.js, since Quokka uses node to do its thing. 💡